back-dispatcher/routers/EverythingRoute/everythingController.ts

```diff
@@ -0,0 +1,33 @@
+import { Request, Response, NextFunction } from "express";
+import dotenv from "dotenv";
+import {
+  exceededFreeTierLimit,
+  mustIncludeAtLeastOneEverything,
+  unknownError,
+} from "../../constants/strings/strings";
+import NewsAPI from "ts-newsapi";
+import { INewsApiEverythingParams } from "ts-newsapi/lib/types";
+import { isOverOneHundred } from "../../utils/limitRequeststoHundred";
+dotenv.config();
+
+export const getEverything = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    const filters = req.body as INewsApiEverythingParams;
```

Comment:
try to use types like this:
 const filters : INewsApiEverythingParams = req.body ;

fix this in all the code